sites/main-site/src/content/blog/2025/survey-results-2025.mdx

```diff
@@ -0,0 +1,296 @@
+
+## General Feedback
+
+So let's start with what the community likes most about nf-core across both users, developers, and user/developers?
+
+<Image
+    src={cloudglobalpositive}
+    class="d-block m-auto"
+    width={600}
+    density={[1.5, 2]}
+    alt="Word cloud with different words corresponding to positive feedback category types as evaluated by the reviewers at different sizes"
+/>
+
+By far the most common feedback was the community 'feel' itself.
+
+Responders often mentioned that the community was just a nice place to be, with people being friendly and helpful (as reflected by other categories such as references to inclusivity, expertise, and speed of responses).
+Furthermore people also appreciated that the diverse number of pipelines are generally of high quality, are reproducible, and the infrastructure around them.
+
+Other appreciated factors were the ease of use of the pipelines and the documentation, as well as the consistency and familiarity of the pipelines (derived from the common template), and the ability to share and re-use components such as nf-core modules in other contexts.
+
+Next let's look what people felt was their biggest difficulties in using or developing within nf-core.
+
+<Image
+    src={cloudglobalimprove}
+    class="d-block m-auto"
+    width={600}
+    density={[1.5, 2]}
+    alt="Word cloud with different words corresponding to improvement feedback category types as evaluated by the reviewers at different sizes"
+/>
+
+Interestingly, by far the largest feedback was documentation - despite other responders saying they really appreciated the documentation!
+There were so many comments on this the reviewers end up splitting this into multiple different categories, and these are still some of the most commented on issues encountered when working with nf-core.
+
+These categories spanned from finding the whole ecosystem very overwhelming to get started with or it being unclear how to get involved (for both users or developers), and setting up nf-core pipelines on infrastructure.
+Another curious discordance with the positive feedback is that while people appreciate the consistency within pipelines due to the common template, people often felt the template was too complex.
+A related feedback was some users finding the pipelines were too large and complicated to get started with.
+
+Other common categories were related to release speed of pipelines and tools/template (typically that they are too fast and there are too many, although sometimes _not fast enough_)
+Communication in terms of how and who decisions within nf-core get made was also sometimes brought up, as well as problems with setting up and debugging the recently implemented nf-test tooling.
+Finally, confusingly a common issue was also Nextflow itself, however the reviewers felt this appears to be either to be due to a misunderstanding that `Nextflow != nf-core`, and the strength of the feelings of Python developers.
```

Comment:
```suggestion
Finally, confusingly a common issue was also Nextflow itself, however the reviewers felt this appears to be either due to a misunderstanding that `Nextflow != nf-core` or the strength of the feelings of Python developers.
```

Reply:
"the strenght of the feelings of Python developers" not sure what you mean by that

Reply:
Made it more explicit :grimacing: